docs/getting-started/transactions/curl/datatx.md

```diff
@@ -490,13 +493,13 @@ curl \
 }
 ```
 
-## (5) Creation, Updation, Deletion of states within a single transaction
+## 4) Creation, Updation, Deletion of states within a single transaction
 
```

Comment:
Updation - replace

Reply:
I will do this at once shot https://github.com/hyperledger-labs/orion-server/issues/284 It has been used in other places too